sagemaker/launch/launch_hyperparameter_tuning.py

```diff
@@ -135,6 +188,17 @@ def run_hyperparameter_tuning_job(args, image, unknownargs):
         "train-yaml-s3": args.yaml_s3,
         "topk-model-to-save": "1",
     }
+
+    # Add strategy-specific parameters
+    if args.strategy == "Hyperband":
+        # Hyperband should control max epochs and early stopping
+        params.update(
+            {
+                "num-epochs": args.hb_max_epochs,
```

Comment:
Can we make max_epochs a general argument of launch_hyperparameter_tuning? So users can define it by themselves.

Reply:
And Hyperband can use the same argument and force users to set it.